imagetest/test_suites/networkperf/setup.go

```diff
@@ -15,7 +16,10 @@ import (
 // Name is the name of the test package. It must match the directory name.
 var Name = "networkperf"
 
+var testFilter = flag.String("networkperf_test_filter", ".*", "regexp filter for storageperf test cases, only cases with a matching name will be run")
```

Comment:
small typo: should probably say "filter for networkperf test cases"

Reply:
Woops, yeah.